tests/test_cssselect.py

```diff
@@ -300,6 +307,8 @@ def css2css(css, res=None):
         css2css(':not(*[foo])', ':not([foo])')
         css2css(':not(:empty)')
         css2css(':not(#foo)')
+        css2css(':is(#bar, .foo)', ':is(#bar, .foo)')
```

Comment:
Is the second argument needed here?

Reply:
I would say so since `:is` is mostly useful when used with more than one argument so the two arguments make sure it covers that case

Reply:
I mean the second argument to `css2css`. It is my impression that it is only needed when both arguments are different, but here both seem to be `':is(#bar, .foo)'`.

Reply:
No it is not required. I must have left it in when a missed space after the comma was causing the test to fail